src/monitor/index.ts

```diff
@@ -0,0 +1,85 @@
+import { config } from "dotenv";
+import { winston, delay, getProvider, getSigner } from "../utils";
+import { AcrossMonitor } from "./AcrossMonitor";
+import { AcrossMonitorConfig } from "./AcrossMonitorConfig";
+import retry from "async-retry";
+import { HubPoolClient } from "../clients";
+import { getDeployedContract } from "../utils/ContractUtils";
+import { SpokePool } from "@across-protocol/contracts-v2";
+
+config();
+
+export async function runMonitor(logger: winston.Logger) {
+  const config = new AcrossMonitorConfig(process.env);
+  // If pollingDelay === 0 then the bot is running in serverless mode and should send a `debug` level log.
+  // Else, if running in loop mode (pollingDelay != 0), then it should send a `info` level log.
+  logger[config.pollingDelay === 0 ? "debug" : "info"]({
```

Comment:
I think you can get this onto 1 line as a result.
```suggestion
  logger.info({
```

Reply:
done